kubernetes/elastic-helm/values.yaml

```diff
@@ -24,38 +30,23 @@ opentelemetry-collector:
         secretKeyRef:
           name: elastic-secret
           key: elastic_apm_secret_token
-  config:
-    exporters:
-      otlp/elastic:
-        endpoint: ${ELASTIC_APM_ENDPOINT}
-        compression: none
-        headers:
-          Authorization: Bearer ${ELASTIC_APM_SECRET_TOKEN}
-    receivers:
-      otlp:
-        protocols:
-          grpc:
-            endpoint: ${MY_POD_IP}:4317
-          http:
-            cors:
-              allowed_origins:
-              - http://*
-              - https://*
-            endpoint: ${MY_POD_IP}:4318
-    service:
-      pipelines:
-        traces:
-          receivers: [otlp]
-          processors: [batch]
-          exporters: [otlp/elastic]
-        metrics:
-          receivers: [otlp]
-          processors: [batch]
-          exporters: [otlp/elastic]
-        logs:
-          receivers: [otlp]
-          processors: [batch]
-          exporters: [otlp/elastic]
+
+  configMap:
+    create: false
+    existingName: elastic-otelcol-agent
```

Comment:
Needed to create a custom configMap as unable to disable components that are not available in our distribution. This is due to a known issue in Helm: https://github.com/helm/helm/issues/12488